docs/use-cases/support.mdx

```diff
@@ -12,6 +13,10 @@ Customer support is an integral part of any organization, serving as the bridge
 XMTP‍ presents a new opportunity to revolutionize customer support by providing encrypted communication channels. By leveraging the features of XMTP, customer support can be taken to new heights, balancing both user convenience and information security.
 
 <div class="chain-list white">
+  <a href="https://www.withblaze.app/" class="chain-item">
+    <img src={blaze} />
```

Comment:
an alt tag would be good here, e.g. "Blaze logo"

Reply:
LOVE - thank you so much, @daria-github - added for all of the logos. I will also be sure to add a pass on the entire site. <3